husky_inekf:
  ros__parameters:
    prior:
      gyroscope_bias: [0.00104398, 0.000137212, -0.000153058]
      accelerometer_bias: [0.00286941, -0.0323794, -0.00220881]
  
  